modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,190 @@
+
+The session handling in ownCloud 10.3.0 has been generally improved, making user and client sessions more stable. 
+If Redis is used for session handling, it is necessary to enable Session Locking to ensure that the mentioned issues no longer occur. 
+
+TIP: You can find out if Redis session handling is configured in your web server if you generate an ownCloud Configreport in the web interface. You will find the value `session.save_handler` set to `redis`. 
+
+* It is recommended to use Redis Session Locking if Redis is used for session handling (minimum required version for `php-redis` is `4.1.0`)
+* Enable Redis Session Locking by setting `redis.session.locking_enabled = 1` in `php.ini`
+
+TIP: If Redis is just used as a memory cache or not in use at all, you do not have to apply changes.
+
+TIP: Please note that Redis Session Locking is not supported in clustered Redis environments.
+
+TIP: If your ownCloud deployment is based on the official Docker images or the Univention appliance, you do not have to apply changes as Redis is not used for session handling (unless you configured it differently using ENV variables).
+
+=== Restructured user/group sharing autocompletion
+
+To cope for long user names or additional user information and to provide a better overview for users, the user/group sharing autocompletion dropdown has been restructured.
+The available information is now distributed vertically to improve space usage and user experience. 
+Screenshots are available in the https://github.com/owncloud/core/pull/35397[pull request]. 
+Other ownCloud clients will align with this behavior with the next releases.
+
+=== SWIFT object storage as primary & secondary storage removed
+
+Following the https://doc.owncloud.com/server/admin_manual/release_notes.html#swift-objectstore-deprecation[deprecation announcement] with ownCloud Server 10.0.9, support for primary and secondary storage via the OpenStack SWIFT protocol has been removed. 
+Please get in contact with ownCloud Support if you're still using OpenStack SWIFT and want to upgrade.
+
+=== S3 object storage as secondary storage is now a separate app
+
+The extension to integrate S3 object storages as secondary storages (`files_external_s3`) has been updated, unbundled from ownCloud Server (was previously part of `files_external`) and released to the https://marketplace.owncloud.com/apps/files_external_s3[ownCloud Marketplace]. 
+If you're using S3 external storage mounts, you have to conduct some steps to ensure continuous operation after upgrading to Server 10.3:
+
+* After the upgrade to Server 10.3 has finished successfully, keep the maintenance mode activated and install/enable `files_external_s3` (either manually or via the Market app) as the app is not bundled with ownCloud Server anymore.
+* After enabling `files_external_s3` (`occ app:enable files_external_s3`) it is necessary to run `occ upgrade` again to conduct config migrations. 
```

Comment:
This step is not necessary! We just tested it. The migration is run during app:enable.
Please remove the second `occ upgrade` step.

Reply:
Shall do.

Reply:
Removed.